src/Storefront/Resources/app/storefront/src/plugin/main-menu/offcanvas-menu.plugin.js

```diff
@@ -60,7 +60,13 @@ export default class OffcanvasMenuPlugin extends Plugin {
                     });
                 });
             });
+            // initialize the plugins again, after Off-Canvas init, otherwise you will miss the JS event listener
+            window.PluginManager.initializePlugins();
         }
+        // re-open the menu if the url parameter is set
+        window.addEventListener('load', () => {
```

Comment:
Is the load event even needed? All JS-plugins are loaded on `DOMContentLoaded` event.